connector_onpay/models/hr_employee.py

```diff
@@ -5,8 +5,8 @@
 
 
 class HrEmployee(models.Model):
-    _inherit = 'hr.employee'
+    _inherit = "hr.employee"
 
     onpay_code = fields.Char(
-        string='OnPay ID',
+        string="OnPay ID",
```

Comment:
If you blackify code, always do that in a separate commit, to make clear the actual code changes